bootstrap/bootstrap.sh

```diff
@@ -0,0 +1,161 @@
+  echo "                              your Kurtosis Lambda code. This is the name of the Docker image that building your Kurtosis Lambda repo will produce."
+  echo "                              This image should not exist yet, as building the Kurtosis Lambda will create it. "
+  echo "                              The image name must match the regex [${ALLOWED_IMAGE_NAME_CHARS}]+ (e.g. 'my-kurtosis-lambda-image')."
+  echo ""
+  exit 1 # Exit with an error so CI fails if this was accidentally called
+}
+
+# =============================================================================
+#                           Arg Parsing & Validation
+# =============================================================================
+lang="${1:-}"
+output_dirpath="${2:-}"
+lambda_image="${3:-}"
+
+if [ -z "${lang}" ]; then
+  echo "Error: Lang cannot be empty" >&2
+  show_help_and_exit
+fi
+if ! grep -q "^${lang}$" "${supported_langs_filepath}"; then
+  echo "Error: Unrecognized lang '${lang}'" >&2
+  show_help_and_exit
+fi
+if [ -z "${output_dirpath}" ]; then
+  echo "Error: Output dirpath must not be empty" >&2
+  show_help_and_exit
+fi
+if [ -d "${output_dirpath}" ] && [ "$(ls -A "${output_dirpath}")" ]; then
+  echo "Error: Output directory '${output_dirpath}' exists, but is not empty"
+  show_help_and_exit
+fi
+if [ -z "${lambda_image}" ]; then
+  echo "Error: Kurtosis Lambda image cannot be empty" >&2
+  show_help_and_exit
+fi
+sanitized_image="$(echo "${lambda_image}" | sed "s|[^${ALLOWED_IMAGE_NAME_CHARS}]||g")"
+if [ "${sanitized_image}" != "${lambda_image}" ]; then
+  echo "Error: Kurtosis Lambda image name '${lambda_image}' doesn't match regex [${ALLOWED_IMAGE_NAME_CHARS}]+" >&2
+  show_help_and_exit
+fi
+
+# =============================================================================
+#                                 Main Code
+# =============================================================================
+# Use language-specific prep script to populate contents of output directory
+if ! mkdir -p "${output_dirpath}"; then
+  echo "Error: Could not create output directory '${output_dirpath}'" >&2
+  exit 1
+fi
+lang_dirpath="${repo_root_dirpath}/${lang}"
+lang_bootstrap_dirpath="${script_dirpath}/${lang}"
+prep_new_repo_script_filepath="${lang_bootstrap_dirpath}/${PREP_NEW_REPO_FILENAME}"
+if ! bash "${prep_new_repo_script_filepath}" "${lang_dirpath}" "${output_dirpath}" "${lambda_image}"; then
+  echo "Error: Failed to prep new repo using script '${prep_new_repo_script_filepath}'" >&2
+  exit 1
+fi
+
+# README file
+output_readme_filepath="${output_dirpath}/${OUTPUT_README_FILENAME}"
+cat <<EOF >"${output_readme_filepath}"
+My Kurtosis Lambda
+=====================
+Welcome to your new Kurtosis Lambda! You can use Example Kurtosis Lambda implementation as a pattern to create your own Kurtosis Lambda.
+
+1. Customize your own Kurtosis Lambda by editing the generated files inside the `/path/to/your/code/repos/kurtosis-lambda/impl` folder
```

Comment:
IMO it's better to have the instructions for customizing the Lambda repo here in the generated README, rather than in the `kurtosis-lambda-starter-pack` README